nsdb-cluster/src/multi-jvm/scala/io/radicalbit/nsdb/cluster/actor/MetadataSpec.scala

```diff
@@ -129,7 +133,7 @@ class MetadataSpec extends MultiNodeSpec(MetadataSpec) with STMultiNodeSpec with
 
   "Metadata system" must {
 
-    "join cluster" in within(10.seconds) {
+    "join cluster" in  {
```

Comment:
same

Reply:
I'm wondering why scalaFmt did not check it for me, nice catch anyway!